Tools2

paolap opened the pull request:
This branch includes Dougie paragraph from this pull_request: https://github.com/ACDguide/BigData/pull/70
in the tools-python1 file instead of the intro.

I also resolved Claire comments, removed the VSCode comment and put Dougie text in a warning box.

I also merged the latest main (i.e. after this pull_request: https://github.com/ACDguide/BigData/pull/68 was merged)

Finally I checked that use of netCDF/NetCDF (capitalise at start of sentence/header) and Zarr was consistent and fixed it where necessary.

Nothing from this branch should generate a conflict, it should just allow us to have a clean slate before restructuring.

dougiesquire reviewed (commented):
Thanks @paolap!

This was pretty hard to review on the Github interface as there're lot's of changes. I'd like to have a proper look through by building this version, but I don't have time to do that today, so I'm just noting a few things for now and I'll have a proper look through early next week.

I'm also planning to have my own stab at reorganising early next week. One difficulty is that you've also added lot's of new material, so I'll need to make sure it's included in whatever I come up with (perhaps I'll try to build off this branch).

For now, here are some comments on this PR:

1. It looks as though there are now duplicated sections in the different computations notebooks. For example, the sections "Common Tasks", "Min / Max / Mean / Stddev" and "Percentiles / Median" seem to appear in both chunking.ipynb and computations.ipynb. This could be ReviewNB misleading me

2. Have the working group goals been moved anywhere, or are you wanting to delete them?

3. Some other minor typos below

dougiesquire commented:
Apologies, I just properly read your first comment @paolap. It's a good idea to get this merged to have a common base for everyone to start thinking about the restructure. Shall we address the comments in my review above, get this merged and then we can all think about reorganising from `main`?

hot007 commented on BigData/data_storage.md:
```diff
@@ -8,28 +8,28 @@ See content navigation in the right hand menu.
 
 ### NetCDF
-[NetCDF](https://www.unidata.ucar.edu/software/netcdf/) is the data format most commonly used in climate science. It is an open data format with full self-described metadata, though it is up to the data creator to ensure the metadata meets any relevant standards, such as the [CF Convention](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html).
+[netCDF](https://www.unidata.ucar.edu/software/netcdf/) is the data format most commonly used in climate science. It is an open data format with full self-described metadata, though it is up to the data creator to ensure the metadata meets any relevant standards, such as the [CF Convention](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html).
```
As this one starts the sentence it probably can stay capitalised?

paolap replied:
I capitalised all of them at the start of the sentences as from Unidata

paolap replied:
it might look like there's lot new from this platform, but I really didn't touch anything out of tools and mostly they are completely new addition rather than replacements. Like these introduction Dougie reviewed

hot007 commented on BigData/tools/tools-python1.md:
```diff
@@ -19,15 +18,19 @@ This is a free, open-source language that is a standard tool used in many organi
-    VSCode is a source code editor which is available for Windows, macOS and Linux. You can edit code locally, or use plugins to remotely connect to servers over SSH. It also integrates with Anaconda, letting you run Python programs in different environments. VSCode is designed to lightweight and adaptable, so has just basic functionalities out of the box and you need to install extensions to add ...
+
+[VSCode](https://code.visualstudio.com)
+    VSCode is a source code editor which is available for Windows, macOS and Linux. You can edit code locally, or use plugins to remotely connect to servers over SSH. It also integrates with Anaconda, letting you run Python programs in different environments. VSCode is designed to lightweight and adaptable, so has just basic functionalities out of the box and you need to install extensions to add ...
```
* is designed to *be* lightweight and 
* Note, git integration in VScode is also really good, but I think that might actually be inbuilt not a plugin now see it works in the browser version in github... so probably fine to not mention it.

paolap commented:
yes, that was the idea, this actually it doesn't have a lot of new additions compared to main.

1) the repetition is because a couple of these notebooks have been created as placeholders from computations. I kept some of the original text to have a reference on how to style them in a comparable way.

2) I removed the goals.md file as it is not used. They are indeed included in the introduction.

3) I fixed the typos you listed, these were some of the few new addition, but the overall tools part needs more reviews anyway

So if you're happy with it I pushed the fixes and I'll merge

hot007 reviewed (approved):
Agree with Dougie, there's a lot here but also it'll help us to all be working from the same basis again so I'm happy to approve this now so we don't hold up further development.

dougiesquire reviewed (approved):
Great - happy to merge

paolap commented:
I also made sure that the typos were fixed in restructure_paola in case so one starts from there, so they won't get lost :-)
